### PR TITLE
Change grib2 file name forecast hour precision from two to three digits

### PR DESCRIPTION
**Pull request by JeffBeck-NOAA**
## DESCRIPTION OF CHANGES: 
Changes forecast hour precision from two to three digits, and adds an "f" prior to the forecast hour in the final grib2 file names.  Requested by GSL.  

Modifies the plot_allvars.py Python script to match this convention.

## TESTS CONDUCTED: 
Tested successfully on Hera with both HRRR3km and HRRR25km domains.

## CONTRIBUTORS (optional): 
@gsketefian

### CODE REVIEW

File: scripts/exregional_run_post.sh [hunk 1]
```diff
@@ -206,9 +206,30 @@ zero exit code."
...
+#
+# A separate ${post_fhr} forecast hour variable is required for the post 
+# files, since they may or may not be three digits long, depending on the
+# length of the forecast.
+#
+#-----------------------------------------------------------------------
+#
+len_fhr=${#fhr}
+if [ ${len_fhr} -eq 2 ]; then
+post_fhr=${fhr}
```
Comment (gsketefian) [inline scripts/exregional_run_post.sh]: Line 220: Indent 2 spaces

File: scripts/exregional_run_post.sh [hunk 2]
```diff
@@ -206,9 +206,30 @@ zero exit code."
...
+elif [ ${len_fhr} -eq 3 ]; then
+  if [ "${fhr:0:1}" = "0" ]; then
+    post_fhr="${fhr:1}"
+  fi
+else
+print_err_msg_exit "\
```
Comment (gsketefian) [inline scripts/exregional_run_post.sh]: Line 226: Indent two spaces (but not lines 227 and 228!)

File: scripts/exregional_run_post.sh (hunk 1 above)
Comment (JeffBeck-NOAA) [reply scripts/exregional_run_post.sh]: Fixed in latest commit.

File: scripts/exregional_run_post.sh (hunk 2 above)
Comment (JeffBeck-NOAA) [reply scripts/exregional_run_post.sh]: Fixed in latest commit.

**Pull request merged**